### ISSUE TITLE
Tab height shrank with upgrade, and the layout mode option is gone

### ISSUE THREAD

**Issue by mvassilevsky** [atom/one-dark-ui#198]
Upon upgrading to 1.10.1, my tabs are significantly smaller, and I don't see the option to revert that change.

Downgrading to 1.9.2 for comparison, it's as if the Compact layout mode were being forced.

**Comment by simurai** [comment]
Yes, the "Spacious" mode got removed in https://github.com/atom/one-dark-ui/pull/197.

In Atom `1.17` there will be [docks](https://github.com/atom/atom/pull/13977) on the side and bottom that have tabs. Since the docks are typically more narrow and have less space, the tabs in this theme got changed to be smaller and more compact. We were considering having different tabs for the center and the docks, but that didn't look right and felt inconsistent.

There are forks like https://atom.io/themes/pristine-ui, but not sure if they will keep the tabs the same.

It's also possible to restyle the tabs in your `styles.less` file. Here just a quick start, but would need some more styling to get closer to the "Spacious" mode version:

```less
// One themes with spacious tabs
.theme-one-dark-ui,
.theme-one-light-ui {

  @import "ui-variables";

  .tab-bar {
    padding-top: @component-padding;
    box-sizing: content-box;
    .tab {
      border-top: 1px solid transparent;
    }
    .tab.active {
      border-top-color: @base-border-color;
    }
  }
}
```

![screen shot 2017-04-12 at 7 41 19 pm](https://cloud.githubusercontent.com/assets/378023/24953910/1da846c4-1fb8-11e7-9bc0-31ecc76b336e.png)

**Comment by mvassilevsky** [comment]
That's unfortunate. Looks like I'll be sticking with 1.9.2, then.

**Comment by simurai** [comment]
> That's unfortunate. Looks like I'll be sticking with 1.9.2, then.

Hopefully `1.9.2` will last for a while without issues. 🙏 

I also liked the spacious tabs, but after using the new docks for a while, having compact tabs made a lot more sense. Once Atom `1.17` is out (next month) or now already in [Beta](https://atom.io/beta), I would encourage to try it out too.

Closing as `wontfix`.

**Comment by havthgem** [comment]
I'm very sad 😭, because I like "Spacious" layout mode very much!
It was very sorrowful to remove the good feature after adding a new feature.
But, Hope "One dark" better.

**Comment by simurai** [comment]
@havthgem Yeah, I hear ya. :worried:

It's just that the "spacious" tabs felt like taking up too much space once used in the new docks. Only two things to offer is using a fork or try to restyle this theme, see ☝️ comment https://github.com/atom/one-dark-ui/issues/198#issuecomment-293540115 for more details.

**Comment by havthgem** [comment]
I saw, I have started trying to adapt to the new layout 😂.

**Comment by tillkruss** [comment]
@simurai: The small tabs are fine on a laptop, but on a 5k display I can hardly see them.

**Comment by ErrOrnAmE** [comment]
Maybe this update could have waited the release of 1.17.
In the meantime, lot of users will have a negative feeling about this change

**Comment by utkarshkukreti** [comment]
@tillkruss I've been using the following to increase the white space since I upgraded to Atom 1.17 today (which included this update I believe):

```less
.theme-one-dark-ui {
  .tab-bar {
    height: 3em;
    .tab {
      line-height: 3em;
      .close-icon {
        top: 0.75em;
      }
    }
  }
}
```

It doesn't look as good as the old version to me, but it's much better than the new one on my 27" 4K display.

**Comment by ErrOrnAmE** [comment]
Oh right, 1.17 got released today, my bad!

**Comment by tillkruss** [comment]
@utkarshkukreti: That works great. Thanks. Just changed `top: 0.75em;` to `top: 0.7em;`.

**Comment by victorwpbastos** [comment]
I change `top: 0.75em` to `top: 0.8em !important` so it apply to modified tabs icon too. And put `right: 0.8em !important` as well.

**Comment by simurai** [comment]
Here another way of "making things bigger":

```
.tab-bar {
  zoom: 1.5;
}
```

It will also affect font size and icons etc. and should work for any theme.

**Comment by JesseLeite** [comment]
Since #209 was closed, I'll comment here as well.

>Those that wanted compact tabs could opt out of spacious. Sad to see this go 😭

![](https://github-atom-io-herokuapp-com.global.ssl.fastly.net/assets/screenshot-main-80d8c9841da6ed11c9d87f31136a4ca9.png)

It's not just about making the tabs bigger @simurai.  Look at the padding around the text, the border-radius around the active tab, and the way the tabs & editor sit a few pixels from the edge of the window.  It's a pretty drastic change; I'm surprised the team didn't start a **_new_** UI theme to go in this new direction, like what was done with `Atom Dark` > `One Dark`.

![](http://b3.ms/gPG9YmNr97JQ)

I hope I'm coming across as respectful/constructive, I just feel like the classic look of One Dark is gone.

**Comment by alexmngn** [comment]
I just played a little bit with the stylesheet, if you wish to increase the height of the tab bar to something that was similar to before, you can open your stylesheet (On Mac, Atom > Stylesheet...), then copy and paste the following:

```
.tab-bar {
	height: 45px;

	li {
		display: flex;
		justify-content: center;
		align-items: center;
	}

	.close-icon {
		top: auto !important;
	}
}
```

Change the value of `height: 45px;` to whatever you prefer.

**Comment by victorwpbastos** [comment]
For now I did `apm install one-dark-ui@1.9.2`. I'm happy again 😃

**Comment by utkarshkukreti** [comment]
@victorwpbastos are you using Atom 1.17? If yes, does it work well with it? (especially with the new Dock feature)

**Comment by victorwpbastos** [comment]
@utkarshkukreti, I'm using Atom 1.17 and everything is working fine!

**Comment by danqing** [comment]
I remember there's something that puts the tabs next to the traffic lights (like what Chrome does). That never really happened, but I feel it's the best way (for the user) to have a spacious tab region without taking up much space.

**Comment by mvassilevsky** [comment]
I've disabled Atom updates and am sticking with Atom 1.15 and One Dark 1.9.1. Both already do everything I want, so updates will only make unwelcome changes to the UI (which, BTW, is pretty typical in the software world now).

**Comment by danielcaldas** [comment]
For a moment I was a bit scared! Thank you @alexmngn now the tabs look just fine :)

**Comment by isteven** [comment]
Sad to see this removed. Bigger tab gives better visibility, not to mention providing some "structure" on the overall look & feel of the editor. Just like a H1 element is bigger compared to P element (not a 100% correct analogy, but well.. )

For now I just make the whole UI font bigger, but I do hope that the spacious tab option will be available again in the next release of One Dark UI.

Cheers :)

**Comment by utkarshkukreti** [comment]
@victorwpbastos thank you!!! It's working great! I'll stick with that version until an Atom update breaks it. :)

**Comment by iparr** [comment]
The following stylesheet additions has put my Atom install in a much more familiar place:
```
// One themes with spacious tabs
.theme-one-dark-ui,
.theme-one-light-ui {

  @import "ui-variables";

  .tab-bar {
    // padding-top: @component-padding;
    height: 3.5em;
    box-sizing: content-box;
    .tab {
      border-top: 1px solid transparent;
      padding-top: .4em;
      padding-left: .1em;
      padding-right: .1em;
    }
    .tab.active {
      border-top-color: @base-border-color;
    }
    .close-icon {
      top: 1em;
    }
  }

  atom-pane .tab.active:before {
    content: "";
    position: absolute;
    pointer-events: none;
    z-index: 2;
    top: 0;
    left: 0;
    bottom: 0;
    width: 2px;
    background: #5497ef;
  }
}
```

**Comment by simonvanderveldt** [comment]
Juding by the comments here it seems like an option to switch between the current compact mode and the previous spacious mode would be really appreciated even though it might "feel" like they are too big for the current docks :)

**Comment by dzift** [comment]
FWIW I would like to see less clutter in my editor. I don't know what docks do but I was perfectly happy and very productive with the layout I had prior to the update. If the spacious layout isn't breaking the docks, instead just makes them look 'worse' (in your opinion) shouldn't that be the users choice to make if they want to change it?

TL;DR Change the default layout, don't remove the options.

**Comment by dkerzig** [comment]
It's not only about the more spacious tabs but the auto-mode! I use synchronized settings on my 13'' MacBook and 5K iMac and I DON'T want to make my tabs more spacious via custom styling on both. 

Please bring back Layout-Mode as opt-in!!!
